db/schema.rb

```diff
@@ -15,6 +15,9 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
```

Comment:
Somehow this got duplicated.  Go ahead and run rails db:migrate on your end to re-generate this schema file.